doc/plans/test_plan.rst

```diff
@@ -0,0 +1,143 @@
+.. _Test_plan:
+
+Zephyr Test Plan
+########################
+
+This document provides a general introduction of the Zephyr Test Plan.
+Zephyr provides test suite for various functionalities offered in
+the project.
+
+**Kernel services and capabilities**
```

Comment:
This will be the section that have the kernel test plan you have in the other file.